Airports/EGHI/SMR/Geo.txt

```diff
@@ -1004,8 +1004,8 @@ N050.46.58.720 W001.50.25.559 N050.46.58.762 W001.50.25.514 smrHHcompass
 ;Stand Lines New
 EGHH Stand Lines (New)            S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00.00.000
 ; 1
-;N050.46.45.032 W001.50.05.735 N050.46.45.258 W001.50.01.646 smrTaxiwayLine
-N050.46.45.144 W001.50.03.763 N050.46.45.258 W001.50.01.646 smrTaxiwayLine
+N050.46.45.032 W001.50.05.735 N050.46.45.258 W001.50.01.646 smrTaxiwayLine
+;N050.46.45.144 W001.50.03.763 N050.46.45.258 W001.50.01.646 smrTaxiwayLine
```

Comment:
This says it's EGHH, are you deliberately changing this? I think this section (and the stuff above for hundreds of lines) might be in the wrong file.

Reply:
I don't think it was changed on purpose

Reply:
Yeah, I have just checked and removing these lines makes no difference to anything, they are from a 'data unverified' commit 10 yrs ago (hence why there's no KMZ too), so given I've removed them I'll just commit that change to this PR (to avoid another issue causing merge conflicts on this file).